includes/wc-term-functions.php

```diff
@@ -305,40 +343,40 @@ function get_woocommerce_term_meta( $term_id, $key, $single = true ) {
  * @return int
  */
 function wc_reorder_terms( $the_term, $next_id, $taxonomy, $index = 0, $terms = null ) {
-
-	if( ! $terms ) $terms = get_terms($taxonomy, 'menu_order=ASC&hide_empty=0&parent=0' );
-	if( empty( $terms ) ) return $index;
+	if ( ! $terms ) $terms = get_terms( $taxonomy, 'menu_order=ASC&hide_empty=0&parent=0' );
+	if ( empty( $terms ) ) return $index;
 
 	$id	= $the_term->term_id;
 
 	$term_in_level = false; // flag: is our term to order in this level of terms
 
-	foreach ($terms as $term) {
+	foreach ( $terms as $term ) {
 
-		if( $term->term_id == $id ) { // our term to order, we skip
+		if ( $term->term_id == $id ) { // our term to order, we skip
 			$term_in_level = true;
 			continue; // our term to order, we skip
 		}
 		// the nextid of our term to order, lets move our term here
-		if(null !== $next_id && $term->term_id == $next_id) {
-			$index++;
+		if (null !== $next_id && $term->term_id == $next_id) {
+			$index ++;
```

Comment:
Very minor and I don't think it's even addressed in the coding standards as far as I can tell, but I don't think I've seen spaces before incrementing or decrementing a var within the rest of the code base.